lib/node_modules/@stdlib/math/base/special/ceil10/package.json

```
@@ -62,5 +62,84 @@
     "round",
     "nearest",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "ceil",
```

```suggestion
      "base_alias": "ceil10",
```

The "base" alias is the alias which is free of dtype information (e.g., `c` and `f` suffixes). The `10` is necessary to understand what the function does and hence should be kept in the base alias.